packages/android-studio/android-studio.pacscript

```diff
@@ -21,7 +21,7 @@ install() {
     cp -r license "$STOWDIR/$name/opt/android-studio/"
     cp -r plugins "$STOWDIR/$name/opt/android-studio/"
     # Creating a launcher in $PATH
-    sudo ln -s "$STOWDIR/$name/opt/android-studio/bin/studio.sh" "/usr/bin/android-studio"
+    ln -s "$STOWDIR/$name/opt/android-studio/bin/studio.sh" "/usr/bin/android-studio"
```

Comment:
This one should probably require `sudo` as we are symlinking to a file owned by `root`